spec/jobs/search/remove_from_index_worker_spec.rb

```diff
@@ -0,0 +1,7 @@
+require "rails_helper"
+require "jobs/shared_examples/enqueues_job"
```

Comment:
this is related to ActiveJob, you can use `workers/shared_examples/enqueues_on_correct_queue` (though you don't need to require it).